manager/controllers/app/plotter_controller.go

```diff
@@ -308,6 +308,7 @@ func (r *PlotterReconciler) setPlotterAssetsReadyStateToFalse(assetToStatusMap m
 	}
 }
 
+//nolint:funlen,gocyclo
```

Comment:
For this and other //nolint comments: can you create an issue with a list of all linting exceptions that were made? Perhaps we can do some refactoring according to the found lint issues.

Reply:
Sure.
FYI: For some of the specific problems on code performance I don't add `//nolint` so it'll be easier to spot.

Reply:
hi @shlomitk1 , we plan to run additional lint fixes.